k256/src/lib.rs

```diff
@@ -16,6 +16,8 @@
 
 #[cfg(feature = "arithmetic")]
 mod arithmetic;
+#[cfg(feature = "arithmetic")]
+mod mul;
```

Comment:
I feel like there's maybe a larger discussion to be had here about refactoring the `arithmetic` module, but we can save that for another PR.